packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -240,8 +257,8 @@ export class KubernetesClient {
     let namespace;
 
     try {
-      const cm = await this.readNamespacedConfigMap(OPENSHIFT_CONSOLE_CONFIG_MAP, OPENSHIFT_CONSOLE_NAMESPACE);
-      if (cm) {
+      const projectGroupSupported = await this.isAPIGroupSupported(PROJECT_API_GROUP);
+      if (projectGroupSupported) {
```

Comment:
below maybe replace .listClusterCustomObject('project.openshift.io' by the name of the constant